### ISSUE TITLE
Conduct an accessibility audit

### ISSUE THREAD

**Issue by thatbudakguy** [sul-dlss/sul-embed#1491]
Had a conversation with the SODA folks today where we did some light a11y testing involving an embedded mirador and the file viewer. It's evident that there are a lot of problems lurking, especially with the file viewer. We probably need to ask for a dedicated audit of all of the various interfaces.

SODA has a [request form](https://app.smartsheet.com/b/form/d100abd9390d40ebbb36e0b47be6d2b2 ) for audits, and if we provide an example of each type of viewer (probably the PURL page) they can evaluate all of them and submit feedback for us to remediate.

**Comment by dbranchini** [comment]
Completed file viewer accessibility review - https://docs.google.com/spreadsheets/d/155sBaCt9dJ3Hy-6mjx3leaCgIDdQ77EhMZabdOLhcRM/edit#gid=634412551. One issue became a redesign of the file viewer. Amy Hodge, Justin Coyne and Andrew Berger and I reviewed. I'm reviewing with Alan for consistency spanning other viewers.

**Comment by lwrubel** [comment]
This issue addresses work on the file viewer alone. There are separate tickets for the other viewers.

**Comment by dbranchini** [comment]
Changed access to the spreadsheet above, so everyone should be able to access it now.

**Comment by dbranchini** [comment]
To summarize the file viewer audit, there weren't too many issues. Most issues reported by SiteImprove (SI) and Axe are false positives/errors because the only way to use these tools is to scan the embedded `iframe` content as a standalone page. Due to that, they throw errors that would be applicable to a standalone page only, such as defining the language in the `html` tag or `header`, `main` and `footer` landmark requirements. In actuality, these are unnecessary because the `iframe` sits within a `main` landmark. I've provided a more detailed summary on the spreadsheet mentioned in the first comment. 

A couple of issues that were problems inspired a redesign of the file viewer - see ticket #.

**Comment by lwrubel** [comment]
Am I interpreting the spreadsheet correctly that there are no accessibility issues for the file viewer requiring work (separate from the redesign work) at this time?

**Comment by lwrubel** [comment]
Closing since @dbranchini confirms there are no issues to be resolved separate from any involved in the redesign work in https://github.com/sul-dlss/purl/issues/801.
